broker-auth-local/src/main/java/org/aktin/broker/auth/apikey/ApiKeyDTO.java

```diff
@@ -0,0 +1,35 @@
+package org.aktin.broker.auth.apikey;
+
+import javax.xml.bind.annotation.XmlRootElement;
+
+/**
+ * Data Transfer Object (DTO) for API key information. This class is used to transfer API key and client DN information between client and server in a
+ * XML format.
+ *
+ * @author akombeiz@ukaachen.de
+ */
+@XmlRootElement(name = "apiKeyCred")
```

Comment:
using lower camelCase for XML not so common. E.g. look at https://en.wikipedia.org/wiki/NIEMOpen which recommends upper CamelCase for elements, native xml typically uses lowercase with hyphens